src/settings-ui/Settings.UI/ViewModels/Flyout/AllAppsViewModel.cs

```diff
@@ -32,125 +32,269 @@ public AllAppsViewModel(ISettingsRepository<GeneralSettings> settingsRepository,
 
             var resourceLoader = Helpers.ResourceLoaderInstance.ResourceLoader;
             GpoRuleConfigured gpo;
-            if ((gpo = GPOWrapper.GetConfiguredAlwaysOnTopEnabledValue()) != GpoRuleConfigured.Disabled && gpo != GpoRuleConfigured.Enabled)
+            gpo = GPOWrapper.GetConfiguredAlwaysOnTopEnabledValue();
```

Comment:
Question: why do these lines stay outside of the code block? (in contrast to _label_ and _icon_ and such)